Makefile

```diff
@@ -188,6 +188,10 @@ openshiftci-presubmit-unittests:
 test-integration:
 	$(RUN_GINKGO) $(GINKGO_FLAGS) tests/integration
 
+.PHONY: test-integration-no-cluster
+test-integration-no-cluster:
```

Comment:
Open question I was wondering: while I understand the value of this, in #5840, we decided to simplify the targets and structure, with a single `test-integration` target that would execute all tests.
So I was wondering if we could still keep a single `test-integration` target, but perhaps instruct Ginkgo to pick the `no-cluster` tests differently. I see something really interesting with [Spec Labels](https://onsi.github.io/ginkgo/#spec-labels) that could be used as a more structured mechanism for organizing and filtering specs.

What do you think?

Reply:
I would prefer to start working on a cluster-less environment from the beginning.

If we have a single test-integration mixing the cluster-full and cluster-less environments, we still need to provide an environment with a cluster, and connect get experience from running a cluster-less env.

Reply:
Okay, that makes sense from an environment setup perspective.

But regarding the Makefile, I'm still wondering, since this means that we need to not forget about running both `test-integration-no-cluster` and `test-integration` targets if we want to run all tests.
So maybe we could keep the `test-integration` target as a single target that would run all tests, but introduce a `test-integration-cluster` that would run the ones requiring a cluster?

Reply:
Yes, that makes sense